### PR TITLE
Update Readme

### LINKED ISSUE #5: Why is private key needed?
From the readme:

> **JWK Content**
>
> The key pair to sign and validate the data structures. This is defined as x5c datastructure which contains the complete key pair.

Is doesn't make sense to pass in the private key. The certificate/public key should be sufficient.

### PR DESCRIPTION
**Pull request by SchulzeStTSI**
Test procedure was splitted into Generation and Validation.

Closes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,15 +69,26 @@ The  JSON Content under RAW is defined as:
    "COSE":**COSE (hex encoded)**,  
    "BASE45": **BASE45 Encoded COMP**, 
    "PREFIX": **BASE45 Encoded Compression with Prefix HC(x):**,  
+   "2DCODE":**BASE64 Encoded PNG**,
    "TESTCTX":{
-               "VERSION":**integer**
-               "SCHEMA":**integer (USED SCHEMA)**, 
+               "VERSION":**integer**,
```
Comment (martin-lindstrom) [inline README.md:74]: The example below holds a string.

Comment (SchulzeStTSI) [reply README.md:74]: Thanks i fixed it.

File: README.md [hunk 2]
```diff
@@ -111,4 +151,4 @@ Contains images of the generated base45 contents(PNG).
 
 ### JWK Content
 
-The key pair to sign and validate the data structures. This is defined as x5c datastructure which contains the complete key pair.
+The key pair to validate the data structure. This is defined as x5c datastructure which contains the public key.
```
Comment (martin-lindstrom) [inline README.md:154]: Suggestion: The public key to validate the data structure. This is defined as an x5c data structure containing the certificate holding this public key.

File: README.md [hunk 3]
```diff
@@ -12,19 +12,39 @@ The test procedure has the following steps:
 2. Apply all test and validation rules to File X (from all countries). 
 3. Fails one rule, the RAW Data File X is highlighted with the related Validation Rule/TestName Fail Status.
```
Comment (martin-lindstrom) [inline README.md:13]: "Fails one rule," -> If one rule fails,

**Review by daniel-eder — APPROVED** [review]

**Pull request merged**